go/hyperkit.go

```diff
@@ -428,7 +430,18 @@ func (h *HyperKit) buildArgs(cmdline string) {
 	}
 
 	for _, p := range h.Disks {
-		a = append(a, "-s", fmt.Sprintf("%d:0,virtio-blk,%s", nextSlot, p.Path))
+		// Default the driver to virtio-blk
+		driver := "virtio-blk"
+		if p.Driver == "" {
+			driver = p.Driver
+		}
+		arg := fmt.Sprintf("%d:0,%s,%s", nextSlot, driver, p.Path)
+
+		// Add on a format instruction if specified.
+		if p.Format != "" {
+			arg += fmt.Sprintf(",format=%s", p.Format)
```

Comment:
I think this can be just `arg += ",format=" + p.Format` and therefore avoid the `fmt.Sprintf`.